utils/src/main/java/io/atomix/utils/serializer/serializers/ByteBufferSerializer.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2016-present Open Networking Foundation
+ *
```

Comment:
Should we change the copyright for the things we added ?

Reply:
I added it as a point in the team meeting, I'm not sure how these things work legally :man_shrugging: